Library/Homebrew/cmd/update.sh

```diff
@@ -95,11 +95,13 @@ read_current_revision() {
 
 pop_stash() {
   [[ -z "$STASHED" ]] && return
-  git stash pop "${QUIET_ARGS[@]}"
   if [[ -n "$HOMEBREW_VERBOSE" ]]
   then
+    git stash pop
     echo "Restoring your stashed changes to $DIR:"
```

Comment:
But then again I guess almost nobody really cares about the verbose output. :wink: